src/hooks/useKeyboardControls.ts

```diff
@@ -13,7 +13,8 @@ export const MovementKeys = {
   enter: ["enter"],
 };
 
-export const MOVEMENT_INTERVAL = 350;
+// TODO: this doesnt seem to be used, can we remove it?
+// export const MOVEMENT_INTERVAL = 350;
```

Comment:
- [ ] [TODO] this doesnt seem to be used, can we remove it?

Reply:
https://github.com/sparkletown/sparkle/pull/840/commits/0fcc08e4ad8d690b9dcd1d9fdeef07ca79f88caa